<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreDigit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>000</string>
    </property>
   </widget>
   <widget class="QLabel" name="challengeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3x5</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(Resultado)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResponse2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResponse1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResponse3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_es">
    <property name="title">
     <string>Ações</string>
    </property>
    <addaction name="actionClearScore"/>
   </widget>
   <addaction name="menuA_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClearScore">
   <property name="text">
    <string>Limpar pontuação</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
